aprs_service_registry/health_checker.py

```
@@ -0,0 +1,453 @@
+                        from aprsd.packets import AckPacket
+
+                        if hasattr(packet, "msgNo") and packet.msgNo:
+                            tx.send(
+                                AckPacket(
+                                    from_call=from_call,
+                                    to_call=packet.from_call,
+                                    msgNo=packet.msgNo,
+                                ),
+                                direct=True,
+                            )
+                    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)